### PR TITLE
Celo LLM Withdraw 

### PR DESCRIPTION
**Pull request by marco-figment**
Simulator Screen Shot - iPhone 13 - 2022-09-06 at 18 54 22

Simulator Screen Shot - iPhone 13 - 2022-09-06 at 18 54 58

Simulator Screen Shot - iPhone 13 - 2022-09-06 at 18 55 57

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+  // Note: if we want to mimick the same behavior as LLD, we'll need MomentJS to handle all date edge cases
+  const getDate = (time: string) => {
+    return new Date(parseInt(time) * 1000).toLocaleDateString();
+  }
+
+  const hasErrors = Object.keys(status.errors).length > 0;
+
+  return (
+      <SafeAreaView style={[styles.root, { backgroundColor: colors.background }]}>
+        <TrackScreen category="WithdrawFlow" name="Amount" />
```
Comment (trentkrogers) [inline apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx]: Nit: Maybe `CeloWithdraw` to be consistent with the other screens?

File: apps/ledger-live-mobile/src/const/navigation.js [hunk 2]
```diff
@@ -231,9 +236,34 @@ export const ScreenName = {
   CosmosClaimRewardsValidationError: "CosmosClaimRewardsValidationError",
   CosmosClaimRewardsValidationSuccess: "CosmosClaimRewardsValidationSuccess",
 
+  OsmosisDelegationStarted: "OsmosisDelegationStarted",
```
Comment (trentkrogers) [inline apps/ledger-live-mobile/src/const/navigation.js:239]: Are these Osmosis-related additions intended?

Comment (marco-figment) [reply apps/ledger-live-mobile/src/const/navigation.js:239]: these were to resolve the merge conflicts reported by Github below, however it doesn't look like Git refreshed the checks.

File: apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx (hunk 1 above)
Comment (marco-figment) [reply apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx]: fixed

**Pull request merged**